packages/oscal-viewer/src/App.js

```diff
@@ -231,8 +220,14 @@ function App() {
 
   return (
     <StyledEngineProvider injectFirst>
+      <link rel="preconnect" href="https://fonts.googleapis.com" />
+      <link rel="preconnect" href="https://fonts.gstatic.com" />
+      <link
+        href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap"
+        rel="stylesheet"
+      />
```

Comment:
Is this really how we should do this? I don't love `<link>` tags in the body like this; in fact, I don't think the `preconnect` will even be honored. Perhaps it needs to go in `public/index.html`? It's static HTML that doesn't need React (and that React doesn't need to care about).

Reply:
My idea here was to keep the top-level styling together (the font, theme, and global styles all being declared right here), but I think your right on it making more sense to have static HTML `link` be in an HTML file. Will move it to the place you suggested.

Reply:
I think this is a good idea, will go ahead and make it a dependency in the project through this method.